lib/SPIRV/SPIRVLowerConstExpr.cpp

```diff
@@ -170,7 +170,7 @@ void SPIRVLowerConstExprBase::visit(Module *M) {
       auto LowerConstantVec = [&II, &LowerOp, &WorkList,
                                &M](ConstantVector *Vec,
                                    unsigned NumOfOp) -> Value * {
-        if (std::all_of(Vec->op_begin(), Vec->op_end(), [](Value *V) {
+        if (std::any_of(Vec->op_begin(), Vec->op_end(), [](Value *V) {
```

Comment:
Do we really need this `if`? Is it possible to have a non-constant value in `ConstantVector`?

Reply:
From commit message and the test it seems `undef` if not a constant value, and can be met inside a `ConstantVector`.

Reply:
I don't know what is the purpose of this `if` though.